src/code/audio_seqplayer.c

```diff
@@ -1195,7 +1193,7 @@ void Audio_SequenceChannelProcessScript(SequenceChannel* channel) {
                     case 0xCB:
                         offset = (u16)parameters[0];
 
-                        scriptState->value = *(offset + scriptState->value + player->seqData);
+                        scriptState->value = *(player->seqData + (u32)(offset + scriptState->value));
```

Comment:
not related to your set of changes, and we dont need to do this now if you dont want to, but at some point we really need to changes all instances of `player->` here to `seqPlayer->` lmao, super confusing with the actual player